dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -2777,7 +2939,9 @@ def test_profiler_factory_class_creates_correct_profiler(self, *mocks):
         data_str = 'test'
         self.assertIsInstance(Profiler(data_str), UnstructuredProfiler)
 
-    def test_save_and_load_structured(self):
+    @mock.patch('dataprofiler.profilers.profile_builder.'
+                'StructuredProfiler._update_chi2')
+    def test_save_and_load_structured(self, *mocks):
```

Comment:
What's happening here? to mock out the tests seems dangerous as it if saving is failing in the tests, it likely will on the real data too. 

Is there a reason we need to mock it out?

Reply:
in clean reports hist/correlations is updated so that it can be tested, does that need to happen for chi_2 too? `clean_report`